### DISCUSSION TITLE
Lets Grow our reach together and connect with the developers

### DISCUSSION THREAD

**Discussion by mehdiiwani** [community/community#62959] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

General

### Body

Greetings of the day GitHub Community!!

I wanted to grow my reach develop new connections with the developers around the world if you are willing you can join my server where we can build a strong community 

https://discord.gg/6tZAG5JnDA

Thankyou to everyone!!

**Comment by Spixmacaw** [comment]
response#2

**Comment by queenofcorgis** [comment]
Hi @mehdiiwani , we’re glad you’re here 😀

Thanks for sharing your Discord with the community. Unfortunately, we currently do not allow self-promotion or advertising on the Community on Discussions. We want to make sure there is space for users to ask questions without overwhelming them with other conversations. Thank you for helping us maintain a productive and tidy community for all our members.
